index.html

```diff
@@ -10,20 +10,33 @@
 			<a class="navbar">Portfolio</a>
 			<a class="navbar">Contact</a>
 		</nav>
-		<div class="about-me">
-			<h1>Welcome to My Page!</h1>
-				<section class="me">	
-					<img class="personal-photo" src="IMG_0368.jpg">
-					<p class="intro">Greetings, my name is Colin Meier and this webpage will be used 						to showcase projects I have completed throughout my time at iO Academy.</p>
-				</section>
-				<section class="info">	
-					<h2>About Me</h2>
-					<p>I was born in New York City and have spent much of my career up until now 						following my passion in wildlife conservation and eco-tourism. In addition to my 						love for animals and ecology I am also a rabid team sports fan, supporting most 						New York based clubs in addition to Borussia Dortmund for European 						football.</p>
-				</section>
-				<section class="photos">
-					<img class="leopard" src="20190909_0624sp.jpg">
-					<img class="bball" src="1244979430.0.jpg">
-				</section>
+			<div class="about-me">
+				<h1>Welcome to My Page!</h1>
+					<section class="me">	
+						<img class="personal-photo" src="IMG_0368.jpg">
+						<p class="intro">Greetings, my name is Colin Meier and this webpage will be used to showcase the work I have completed throughout my time at iO Academy.</p>
+					</section>
+					<section class="info">	
+						<h2>About Me</h2>
+						<p>I was born in New York City and have spent much of my career up until now following my passion in wildlife conservation and eco-tourism. In addition to my love for animals and ecology I am 							also a rabid team sports fan, supporting most New York based clubs in addition to Borussia Dortmund for European football.</p>
+					</section>
+					<section class="photos">
+						<img class="leopard" src="20190909_0624sp.jpg">
+						<img class="bball" src="1244979430.0.jpg">
+					</section>
+				</div>
+				<div class="portfolio">
+						<h2>Portfolio</h2>
+						<p class="projects">These are a few of the projects I have completed during the program.</p>
+					<section class="box-one">
+						<p class="project-description">This is where I will post my work!</p>
+						<a class="git-link" href="https://github.com">GitHub</a>
```

Comment:
Maybe add a link to take you to a new window to keep user interaction easier to get back to your page?